### DISCUSSION TITLE
The debug executable does not exist

### DISCUSSION THREAD

**Discussion by nadirion** [leap71/PicoGK#37] [General]
Hi, Josefine.
I'm new to Visual Studio. Opened PicoGK Example.csproj and pressing F5, it gives me an error: The debug executable 'C:\Users\Nadir\Documents\PicoGK Example\bin\Debug\net8.0\PicoGK Example.exe' specified in the 'PicoGK Example' debug profile does not exist. Have I missed something?

**Comment by LinKayser** [comment]
Maybe try if you can run an empty C# app. Here's a description.

https://github.com/leap71/PicoGK/blob/main/Documentation/VisualStudio_CreateConsole.md

Best,
Lin
